base/setup/usetup/usetup.c

```diff
@@ -27,6 +27,7 @@
 
 #include <usetup.h>
 #include <math.h>
+#include <ntstrsafe.h>
```

Comment:
Include not needed since the header is already present in usetup.h

Reply:
include what you use is a good practice.

Reply:
Definitively @learn-more , but here you are including something that has been already included via usetup.h.